desktop-app/app/components/Toolbar/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+
+/**
+ * Application toolbar that appears at the top of a window.
+ */
+const Toolbar = () => {
```

Comment:
@sidthesloth92 Can we name it something like a Horizontal Spacer? As it is not technically a toolbar.

Reply:
Totally agreed..😭

Reply:
Done.